CONTRIBUTING.rst

```diff
@@ -82,3 +82,20 @@ coverage rate should increase or stay the same. Automatic services will ensure
 your code works on all the operative systems and package combinations
 poliastro support - specifically, note that poliastro is a Python 3 only
 library.
+
+Development environment
+-----------------------
+
+To set up a development environment:
+
+1. `Install git <https://git-scm.com/>`_ on your computer.
+2. `Register to GitHub <https://github.com/>`_.
+3. `Fork poliastro <https://help.github.com/articles/fork-a-repo/>`_.
+4. `Clone your fork <https://help.github.com/articles/cloning-a-repository/>`_.
+5. Install it in development mode using
+   :code:`pip install --editable /path/to/poliastro` (this means that the
+   installed code will change as soon as you change it in the download
+   location).
+6. Make changes and commit.
```

Comment:
I would create a branch first, otherwise people are going to get confused with `master` branches everywhere.